e2e/specs/ward-to another-ward.spec.ts

```diff
@@ -0,0 +1,81 @@
+// specs/ward-allocation.spec.ts
```

Comment:
Spaces are not permitted in file names. Let's rename this to `ward-allocation.spec.ts`.